refextract/references/engine.py

```diff
@@ -626,12 +628,12 @@ def look_for_hdl_urls(citation_elements):
 # End of elements transformations
 
 def print_citations(splitted_citations, line_marker):
-    print('* splitted_citations')
-    print('  * line marker %s' % line_marker)
+    LOGGER.debug(u'* split_citations')
+    LOGGER.debug(u"  * line marker %s", line_marker)
```

Comment:
why here we keep the ``*``?

Reply:
I know it looks weird, I see that whoever wrote this used the * to sort of format the response and I thought maybe it was put for a purpose. Should I remove it?

Reply:
it was done to create a hierarchy through indentation (so a logging message from a deeper level in the call stack gets indented more).